bin/physlr-make

```diff
@@ -192,6 +196,11 @@ help:
 version:
 	@echo "physlr v0.2.0"
 
+################################################################################
+# Test m=0 prune_branches=0 prune_bridges=0
+test: \
```

Comment:
Does the test not work with m>0, or doing pruning? The downside of this is that we won't be testing any of the code working with those parameters

Reply:
I could change `m` to something else and change the test a bit. No clue about `prune_branches` though. Could try 1 and see if that would still give me an aouput

Reply:
Ideally the small test would be as representative as possible of a typical physlr run

Reply:
Well the intention was to test if the code would run properly, so I think as long as it does executes the code that would have ran in a typical physlr run, it will be fine. So I will change the test set to accommodate m greater than 0 but the other parameter is little harder to work around.